layouts/partials/meta.html

```diff
@@ -3,6 +3,8 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
 <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+<meta property="og:title" content="Digital Digital">
```

Comment:
Make this dynamic based on the page. See `<title>`. You could also add `og:description` while you're there.